includes/class-comment.php

```diff
@@ -66,6 +70,7 @@ public static function register_comment_types() {
 			array(
 				'label'       => __( 'Bookmark', 'webmention' ),
 				'description' => __( 'A bookmark (or linkblog) is a post that is primarily comprised of a URL, often title text from that URL, sometimes optional text describing, tagging, or quoting from its contents.', 'webmention' ),
+				'icon' => '🔖',
```

Comment:
https://developer.wordpress.org/resource/dashicons/#book

Reply:
maybe use https://developer.wordpress.org/resource/dashicons/#tag instead because of "read" activity